<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aplicativo Hipertermia</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="imagenes_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="imagen_termica_layout"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
      <property name="rightMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QLabel" name="parametros_termicos_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Parametros termicos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temperatura_maxima">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Temperatura maxima:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temperatura_maxima_value">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>43 °C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temperatura_minima">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Temperatura minima:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temperatura_minima_value">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>30 °C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temperatura_promedio">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Temperatura promedio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temperatura_promedio_value">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>35 °C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="control_temperatura_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Control de temperatura</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="toggleControl">
          <property name="text">
           <string>Control</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="toogleLazoAbierto">
          <property name="text">
           <string>Lazo Abierto</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="actuador_label">
        <property name="text">
         <string>Actuador</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QSpinBox" name="actuado_value"/>
        </item>
        <item>
         <widget class="QPushButton" name="actuador_value_send">
          <property name="text">
           <string>Enviar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="temperatura_maxima_deseada_label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Temperatura maxima deseada</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="temperatura_maxima_deseada_value"/>
      </item>
      <item>
       <widget class="QLabel" name="temperatura_minima_deseada_label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Temperatura minima deseada</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="tempertatura_minima_deseada_value"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="control_P_label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>P</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="control_P_value"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="control_I_label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>I</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="control_I_value"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="control_D_label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="control_D_value"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="send_button">
        <property name="text">
         <string>Enviar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="toogleCSV">
        <property name="text">
         <string>Tomar datos CSV</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
